ngfizzy/ps-notes
# Fetch Notes

### WHAT DOES THIS PR DO?

This pr implements fetching of notes and rendering notes from the database

### HOW CAN WE MANUALLY TEST THIS?

* Login to your dashboard
* If you have any data in your api database, you should see them in the dom
* If none, then create a new note. It should appear on the dashboard.
